services-camel/src/test/java/edu/unc/lib/dl/services/camel/BinaryEnhancementProcessorTest.java

```diff
@@ -101,11 +116,23 @@ public void testExistingUriHeader() throws Exception {
         verify(message, never()).setHeader(FCREPO_RESOURCE_TYPE, Binary.getURI());
     }
 
+    @Test
+    public void testEditThumbnail() throws Exception {
```

Comment:
I think this test is out of date? It should probably just be "testNonBinary". I'm not sure the `setMessageBody("image/*");` bit is doing anything either?

Reply:
Looks like it needs the message body, otherwise the processor will give a NPE